src/tokens/transformation/skia/index.ts

```diff
@@ -46,6 +46,30 @@ const getSortKey = ({ name, value }) => {
   // Its a color, but not a primitive
   return sortOrder.length
 }
+
+const dynamicPalettePrimitives = [
+  'primary',
+  'secondary',
+  'tertiary',
+  'neutral',
+  'neutralVariant',
+  'error'
+]
+const maybeDynamicProps = (token: TransformedToken) => {
+  if (!token.name.includes('primitive')) return
```

Comment:
Sorry if I'm totally confused here, but shouldn't this return if the name _does_ include `primitive` as primitives are not dynamic?

Reply:
I've renamed this function to `maybeMaterialColorProps` which is a better name - what this function is doing is mapping the Material color primitives to their equivalent from the Chromium color mixer.